android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -283,6 +360,19 @@ class RowndClient constructor(
     }
 
     fun requestSignIn() {
+        var isAppConfigLoading = isAppConfigLoadingWithCallback {
```

Comment:
What do you think about putting all of this very similar logic into the `displayHub()` method?

Reply:
Update has been made